src/controllers/userController.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express";
+import { v4 as uuidv4 } from "uuid";
+import bcrypt from "bcrypt";
+import User from "../models/userModel";
+
+export const createUser = async (req: Request, res: Response) => {
+	try {
+		const { firstName, lastName, email, password } = req.body;
```

Comment:
Please add a middleware the validate these fields. I remember we had one on the last project.